app/src/main/java/me/saket/dank/ui/user/UserSessionRepository.java

```diff
@@ -49,6 +63,24 @@ public void removeLoggedInUsername() {
     loggedInUsername.set(EMPTY);
   }
 
+  public Completable switchAccount(String username, Context ctx) {
+    try {
+      if (username == null) {
```

Comment:
You should mark username as `@Nullable` in parameter list if it really can be null. Otherwise this check is not needed.